common/djangoapps/student/views.py

```diff
@@ -643,6 +641,18 @@ def is_course_blocked(request, redeemed_registration_codes, course_key):
 
     return blocked
 
+
+def is_coach_on_master_course(user, course):
+    db_query = 'SELECT coach_id FROM ccx_customcourseforedx WHERE coach_id = %s AND course_id= %s;'
```

Comment:
Write this as CustomCourseForEdx.objecs.filter(coach=user, course_id=str(course.id)).count() > 0 which returns True or False depending if user is coach on the master course.